aeron-archive/src/main/java/io/aeron/archive/ArchiveConductor.java

```diff
@@ -80,6 +80,7 @@
     private volatile boolean isAbort;
 
     private final RecordingSummary recordingSummary = new RecordingSummary();
+    private final RecordingSummary otherRecordingSummary = new RecordingSummary();
```

Comment:
Probably better to have this scoped to the call for simplicity.

Reply:
:+1: Fixed in latest commits.